c/tests/test_tables.c

```diff
@@ -4812,38 +4942,47 @@ test_ibd_finder_multiple_trees(void)
     tsk_size_t j, k;
     tsk_treeseq_t ts;
     tsk_table_collection_t tables;
-    tsk_id_t samples[] = { 0, 1, 0, 2 };
-    tsk_id_t *pair;
+    tsk_id_t samples[] = { 0, 1, 2 };
+    tsk_id_t pairs[][2] = { { 0, 1 }, { 0, 2 } };
+    tsk_size_t num_samples = 3;
     tsk_size_t num_pairs = 2;
     tsk_ibd_result_t result;
-    double true_left[2][2] = { { 0.0, 0.7 }, { 0.7, 0.0 } };
-    double true_right[2][2] = { { 0.7, 1.0 }, { 1.0, 0.7 } };
+    double true_left[2][2] = { { 0.0, 0.75 }, { 0.75, 0.0 } };
```

Comment:
In case anyone is wondering, this is for 32 bit tests. 0.7 isn't a round number, and it's troublesome for comparing equality on some hardware.